### ISSUE TITLE
Quick panel \ref with files using the \import command from the import package

### ISSUE THREAD

**Issue by watkinrt** [SublimeText/LaTeXTools#499]
The import LaTeX package allows you to use complicated file trees (which is particularly helpful when writing a book or thesis). In the current implementation of Quick panels with the \ref command, only \include and \input commands are searched for. I've come up with a solution to additionally search for \import commands and then search through their file tree appropriately. At line 81 of latex_ref_completions.py is the following code:

``` python
# search through input tex files recursively
    for f in re.findall(r'\\(?:input|include)\{([^\{\}]+)\}', src_content):
        find_labels_in_files(rootdir, f, labels)
```

To also search for import commands (which need to be parsed somewhat differently), you can add code like this

``` python
importLines = re.findall(r'\\(?:import)[*]?\{([^\{\}]+)\}\{([^\{\}]+)\}', src_content)
    for (base,f) in importLines:
        find_labels_in_files(rootdir+"/"+base, f, labels)
```

after the above code. I've implemented this and it works for the cases I'm interested in; however, I haven't check to see how robust it is.

**Comment by schinknbrot** [comment]
Nice one!

Thank you a lot!

**Comment by ig0774** [comment]
Fixed in v3.13.0
